scripts/git.sh

```
@@ -0,0 +1,204 @@
+        echo "[SUCCESS] Git の core.excludesfile が正しく設定されています"
+        return 0
+    else
+        echo "[ERROR] Git の core.excludesfile が $config_value になっています"
+        return 1
+    fi
+}
+
+# Gitコマンドの検証
+verify_git_command() {
+    if ! command -v git; then
+        echo "[ERROR] "gitコマンドが見つかりません""
```

Shellcheck (0.10.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)